ValheimExportHelper/Resources/CreateAssetBundles.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEditor;
+using System.IO;
+
+public class CreateAssetBundles
+{
+    [MenuItem("Valheim/Build AssetBundles")]
+    static void BuildAllAssetBundles()
+    {
+        string assetBundleDirectory = "Assets/AssetBundles";
+        if(!Directory.Exists(assetBundleDirectory))
```

Comment:
Shouldn't need to test for this, I think CreateDirectory will work fine if it already exists.